pkg/plugins/helm/v1/init.go

```diff
@@ -144,31 +146,21 @@ func addInitCustomizations(projectName string) error {
 	// by https://github.com/kubernetes-sigs/kubebuilder/pull/2119
 
 	// Add leader election arg in config/manager/manager.yaml and in config/default/manager_auth_proxy_patch.yaml
-	err := util.InsertCode(managerFile,
+	err := kbutils.InsertCode(managerFile,
 		"--leader-elect",
 		fmt.Sprintf("\n        - --leader-election-id=%s", projectName))
 	if err != nil {
 		return err
 	}
-	err = util.InsertCode(filepath.Join("config", "default", "manager_auth_proxy_patch.yaml"),
+	err = kbutils.InsertCode(filepath.Join("config", "default", "manager_auth_proxy_patch.yaml"),
 		"- \"--leader-elect\"",
 		fmt.Sprintf("\n        - \"--leader-election-id=%s\"", projectName))
 	if err != nil {
 		return err
 	}
 
-	// Increase the default memory required.
-	err = util.ReplaceInFile(managerFile, "memory: 30Mi", "memory: 90Mi")
```

Comment:
Is this no longer necessary?

Reply:
Yeah, with this kb [commit](https://github.com/kubernetes-sigs/kubebuilder/pull/2255/files) our defaults are already higher now.